deployment/bin/README.md

```diff
@@ -0,0 +1,3 @@
+This directory is intended to be mounted in a running docker image under
+/kb/deployment/bin so that it's contents can be used to build and store Go
```

Comment:
it's -> its, and Go binaries? Are you sure this is needed?

Reply:
The entrypoint script goes here as well. The original readme is way out of date and I've fixed it.